src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { FC, ReactNode, ReactElement } from 'react';
+import { GuidanceState, DocumentationResource, State } from 'pathways-model';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import classes from './ExpandedNode.module.scss';
```

Comment:
According to style guide, CSS Module styles should be imported as `styles` so rename `classes` -> `styles`.

Reply:
Can you point out where this is in the style guide? I can't find it and I noticed DropDown, Header, Navigation, Node, PathwayList, and PatientSnapshot use `classes` instead of `styles`. It seems like only Arrow uses `styles`

Reply:
https://github.com/mcode/pathways/wiki/Code-Style-Guide#importing-styles

yeah I think we created the code style guide afterwards and never went back and changed the older files... but for new files we should probably follow the guide.

Reply:
Changed `classes` --> `styles` and `indexClasses` --> `indexStyles`
Updated in all files in PR 11 https://github.com/mcode/pathways/pull/11